<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMindMapDtIconTool</class>
 <widget class="CDiagramTool" name="CMindMapDtIconTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Icon</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
     <item>
      <widget class="QToolButton" name="m_btnSelectIcon">
       <property name="toolTip">
        <string>Add Icon</string>
       </property>
       <property name="text">
        <string>Add Icon</string>
       </property>
       <property name="icon">
        <iconset resource="../../res/mac/res.qrc">
         <normaloff>:/res/icons/node-icon.png</normaloff>:/res/icons/node-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btnDeleteAllIcons">
       <property name="toolTip">
        <string>Delete All Icons</string>
       </property>
       <property name="text">
        <string>Delete All Icons</string>
       </property>
       <property name="icon">
        <iconset resource="../../res/mac/res.qrc">
         <normaloff>:/res/icons/node-icon-delete.png</normaloff>:/res/icons/node-icon-delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
     <item>
      <widget class="QToolButton" name="m_btnSelectPixmap">
       <property name="toolTip">
        <string>Set Pixmap</string>
       </property>
       <property name="text">
        <string>Pixmap</string>
       </property>
       <property name="icon">
        <iconset resource="../../res/mac/res.qrc">
         <normaloff>:/res/icons/node-pixmap.png</normaloff>:/res/icons/node-pixmap.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_btnDeletePixmap">
       <property name="toolTip">
        <string>Delete Pixmap</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../res/mac/res.qrc">
         <normaloff>:/res/icons/node-remove.png</normaloff>:/res/icons/node-remove.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CDiagramTool</class>
   <extends>QWidget</extends>
   <header>core/CDiagramWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/mac/res.qrc"/>
  <include location="../../res/mac/res.qrc"/>
 </resources>
 <connections/>
</ui>
